base/strings/string.jl

```diff
@@ -231,12 +231,22 @@ end
     end
     trailing = utf8_trailing[b + 1]
     if l < i + trailing
-        return '\ufffd', i+1
-    end
-    c::UInt32 = 0
-    @inbounds for j = 1:(trailing + 1)
+        for j in (i+1):l
+            @inbounds if !is_valid_continuation(codeunit(s, j))
+                return '\ufffd' j
```

Comment:
comma is missing here?

Reply:
thank you for the catch! I will fix it is decided that this should be merged (for now I feel that the proposal of @StefanKarpinski in #24420 will make it redundant).